src/gccrs.rs

```diff
@@ -136,7 +136,7 @@ impl Gccrs {
     }
 
     /// Convert arguments given to `rustc` into valid arguments for `gccrs`
-    pub fn handle_rust_args(args: &[String]) -> Result {
+    pub fn compile_rust_args(args: &[String]) -> Result {
```

Comment:
Now the method name sounds like it compiles _the arguments_. Why not just "`compile(args)`"?

Reply:
I have a bit of an issue with `compile` because it sounds like we're simply using some sort of gccrs wrapper, which would logically take gccrs arguments only. I would be okay with `compile` since the gccrs module will not become its own crate or anything, but I feel weird looking at `Gccrs::compile(args)` when the args in question are clearly `rustc` args and not `gcc` args.

I don't know, I thought this was a good compromise but I'm okay with renaming it as just `compile` if you'd like

Reply:
That's a good point. What about the name `compile_with_rust_args`? So adding a "with". (This is really NIT picking, so if you don't want to change it, but just merge it, I'd also be ok with that)

Reply:
That's a good point. What about the name `compile_with_rust_args`? So adding a "with". (This is really NIT picking, so if you don't want to change it, but just merge it, I'd also be ok with that)

Reply:
Perfect! This is what I was thinking too this morning :)